app/jobs/close_patients_job.rb

```diff
@@ -30,8 +30,14 @@ def perform(*_args)
         patient[:monitoring_reason] = 'Completed Monitoring (system)'
       end
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if patient.save! && patient.email.present? && patient.self_reporter_or_proxy?
+      # Send closed email or SMS to patient if they are a reporter
+      if patient.save! && patient.self_reporter_or_proxy?
+        if ['SMS Texted Weblink', 'SMS Text-message'].include? patient.preferred_contact_method
+          PatientMailer.closed_sms(patient)
```

Comment:
Could you do a `.deliver_later(wait_until: patient.time_to_contact_next)` here, call `TwilioSender.send_sms` from within `closed_sms`  and get rid of the need to have the send_sms_job?

Reply:
Does that work with non-emails? From the docs it looks like it's only for email objects https://api.rubyonrails.org/classes/ActionMailer/MessageDelivery.html

I can try this later today to see if this method works

Reply:
I think it should work because I don't think that the PatientMailer knows anything about the jobs it is enqueueing WRT what is actually happening in the PatientMailer method that is being called whether it be a call to `mail()` or a call to our `TwilioSender`

Reply:
now using deliver_later